fastlane/lib/fastlane/documentation/actions_list.rb

```diff
@@ -44,100 +42,119 @@ def self.print_all(platform: nil)
 
     def self.show_details(filter: nil)
       puts "Loading documentation for #{filter}:".green
-
       puts ""
 
-      all_actions do |action, name|
-        next unless name == filter.strip
+      action = find_action_named(filter)
 
-        rows = []
-        rows << [action.description] if action.description
-        rows << [' ']
-        if action.details
-          rows << [action.details]
-          rows << [' ']
+      if action
+        unless action < Action
+          UI.user_error! action_subclass_error(filter)
```

Comment:
Given that there is improperly defined action:

When the user asks to see all of the actions, I think it makes sense that we display an error, but then also show all the other information that we can in the table. This was true before, and is still true with this change.

When the user is asking to see the details about the improperly defined action, the previous behavior crashed with a stack trace. These changes make it so that we display a helpful message and "crash" (so not a change in behavior with regard to success/failure).

I think it is helpful to stop executing (rather than showing the whole list) in this case, because the user asked for something specific, and we know what they're referring to, but it is broken and can't be displayed.